src/spac/transformations.py

```diff
@@ -261,3 +262,137 @@ def rename_observations(adata, src_observation, dest_observation, mappings):
         )
 
     return adata
+
+
+def normalize_features(
+    adata: anndata,
+    low_quantile: float = 0.02,
+    high_quantile: float = 0.02,
+    target_layer: str = None,
+    new_layer_name: str = "normalized_feature",
+    overwrite: bool = True
+):
+
+    """
+    Normalized the features of a dataset stored in an AnnData object.
+    Any entry lower than the value corresponding to low_quantile of the column
+    will be assigned a value of 0, and entry that are greater than
+    high_quantile value will be assigned as 1. Other entries will be normalized
+    with (values - quantile min)/(quantile max - quantile min).
+    Resulting column will have value ranged between [0, 1].
+
+    Parameters
+    ----------
+    adata : anndata.AnnData
+        An AnnData object containing the data to be normalized.
+
+    low_quantile : float, optional (default: 0.02)
+        The lower quantile to use for normalization. Determines the
+        minimum value after normalization. Must be a positive float.
+
+    high_quantile : float, optional (default: 0.02)
+        The higher quantile to use for normalization. Determines the

```

Comment:
positive float ]0,1]

Reply:
The high quantile should be (0,1] as low can be [0, 1), high must be greater than low